vllm/model_executor/layers/sampler.py

```diff
@@ -149,11 +150,15 @@ def _apply_logits_processors(
         logits_processors = sampling_params.logits_processors
         if logits_processors:
             found_logits_processors = True
+            logits_processor_argc = [len(fn_signature(fn).parameters) for fn in logits_processors]
             for seq_id in seq_ids:
                 logits_row = logits[logits_row_idx]
                 token_ids = sampling_metadata.seq_data[seq_id].output_token_ids
-                for logits_processor in logits_processors:
-                    logits_row = logits_processor(token_ids, logits_row)
+                for i, logits_processor in enumerate(logits_processors):
```

Comment:
**Thought**

We could remove this, since there is no need to pass the `seq_id` anymore, right?

Reply:
I believe we still need to support passing the seq_id in for outlines' logits processors.

Reply:
Outlines has accepted a PR that removes this need: https://github.com/outlines-dev/outlines/pull/628

Reply:
I see, I'll revert the change then

Reply:
don't forget to revert the change!